.circleci/config.yml

```diff
@@ -124,8 +124,8 @@ commands:
                 # For whatever reason we have to install pytorch first. If it isn't
                 # it installs the 1.4 cpuonly version. Which is no good.
                 conda install -q -y pytorch torchvision cudatoolkit=11.3 -c pytorch
-                conda install -q -y -c conda-forge ninja numpy pytest pytest-cov ccache hypothesis
-                pip install pytest-sugar pytest-xdist pytest-benchmark opencv-python cython
+                conda install -q -y -c conda-forge ninja numpy pytest pytest-cov ccache hypothesis pytest-mock psutil
+                pip install pytest-sugar pytest-xdist pytest-benchmark opencv-python cython mock pygame
```

Comment:
Curious, what is pygame used for? I thought the GUI viewers got all remade on top of Magnum

Reply:
There is a test that tests the pygame functionality for human render method on the gym wrapper. If pygame is not present, it should just not run the test I believe